MEETINGS.md

```diff
@@ -1,32 +1,27 @@
-# GitOps Working Group Meetings
-The GitOps Working group typically meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).
-[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+# Meetings
 
-## Upcoming Meetings
-### Thursday, April 15th at 18:00 GMT
-- [Video Conference Link](https://zoom.us/my/cncfgitopswg?pwd=aE9FMjR0M1djd3huMlNUMWZXSXpyQT09)
-- Meeting Passcode: `77777`
+The GitOps Working Group typically meets on the second Thursday of each month at 18:00 GMT, and are listed below. This meeting is also on the [CNCF calendar](https://www.cncf.io/calendar/).
 
-## Past Meetings
-### Thursday, March 11th at 18:00 GMT
+Additionally Working Group Committees meet as needed to complete their projects, and are also listed below.
 
-- [Video Recording](https://zoom.us/rec/share/SYMUyXzmVF4eLmO9d2N3AOQ3ror5cJo361pnz_PtxUnDMio0vXRd-qpjqMHPnGeo.J_6sc5Mv8IIKi9jr?startTime=1615486031000)
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+See the [Google Doc](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for detailed join info, agenda, and notes for all GitOps Working Group meetings.
 
-### Thursday, February 11th at 18:00 GMT
+## Working Group Monthly
 
-- [Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)
-- Passcode: `fwPy8^fb`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+- May 13, 2021: [join link and agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.lq3s56hyydm0)
+- April 15, 2021: [recording](https://www.youtube.com/watch?v=c_GqwvP5Wqw&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=5)
+- March 11, 2021: [recording](https://www.youtube.com/watch?v=P5Ib1CubO54&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=4)
+- February 11, 2021: [recording](https://www.youtube.com/watch?v=NilaC6Jhq_8&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=3)
+- January 14, 2021: [recording](https://www.youtube.com/watch?v=JypiRn8HTbw&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=2)
+- December 10, 2020: [recording](https://www.youtube.com/watch?v=LnzIE6tDfbQ&list=PLXOML2VBdIo4-biBmCbfNkP0ywF0f5mau&index=1)
 
-### Thursday, January 14th at 17:00 GMT
+## GitOps Principles Committee Meetings
 
-- [Video Recording](https://weaveworks.zoom.us/rec/share/_qrMRze16IHb2d4KIOHSa2Wxb_XzbU41-ZH6YZI5aJkUqOHM1bBQLCQhkFva5xo0.ay0LdxTVPCTlgbSd)<br>
-- Passcode: `EpQ26^cV`
-- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+- April 14, 2021: [recording](https://www.youtube.com/watch?v=zWxAZuHNGYM&list=PLXOML2VBdIo6XfUTaIbanBN2fIDEyR25s&index=3)
```

Comment:
For example:
• Are there Chat Messages for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) from participants who didn't raise their hands to speak?
• "+1" can appear in Chat Messages.
• When you time permits, can Zoom recording for [GitOps Principles Committee - April 14, 2021](https://www.youtube.com/watch?v=zWxAZuHNGYM) be added to [Google Doc: April 14, 2021]( https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.7wmerton06if)?

Thanks @scottrigby

Reply:
Yes that is annoying. The goal here was to make the meetings more easily accessible, in a consistent format. I agree the zoom text chat is an important part of the conversation, and should not be left out.

I was able to download all the zoom chat text files separately, and could link to each in the YouTube video description. We could host the text files themselves in this repo, in a `meetings` directory. A slightly more organized version of https://github.com/gitops-working-group/gitops-working-group/issues/9. I'll update this PR

Reply:
OK added text chat files, and linked from MEETINGS.md. Once merged, we can also link to these from the YouTube video description for each.

Reply:
From [Google Doc: December 10, 2020](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.e0ignly90e17):
> [Omer] Chat can be saved in new versions of zoom, Scott will save

-----

@mjpitz [asked:](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit?disco=AAAAHdlWQWw)
> Is this available anywhere?

Yes.  It's available at:
https://github.com/scottrigby/gitops-working-group/blob/meetings-youtube/meetings/monthly/2020-12-10-chat.txt
and
https://weaveworks.zoom.us/rec/share/jyagKQsL7irvi1pF-RHzrrwH-Sqa59wNnBK0F2H8QYBo_rxJZOXMO-5j5CwUgKiK.9cl0var5BN-z0qrI
Passcode: `%mi$&f8e`

@mjpitz Thanks for asking about meeting chats.

@scottrigby Thanks for publishing them.